components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -106,18 +106,20 @@ void P3Microphysics::initialize_impl (const util::TimeStamp& t0)
   // Note: Some variables in the structures are not stored in the field manager.  For these
   //       variables a local view is constructed.
   const Int nk_pack = ekat::npack<Spack>(m_num_levs);
-  auto pmid  = m_p3_fields_in["p_mid"].get_reshaped_view<const Pack**>();
-  auto T_atm = m_p3_fields_out["T_mid"].get_reshaped_view<Pack**>();
-  auto ast   = m_p3_fields_in["cldfrac_tot"].get_reshaped_view<const Pack**>();
-  auto zi    = m_p3_fields_in["z_int"].get_reshaped_view<const Pack**>();
+  auto pmid           = m_p3_fields_in["p_mid"].get_reshaped_view<const Pack**>();
+  auto pseudo_density = m_p3_fields_in["pseudo_density"].get_reshaped_view<const Pack**>();
+  auto T_atm          = m_p3_fields_out["T_mid"].get_reshaped_view<Pack**>();
+  auto ast            = m_p3_fields_in["cldfrac_tot"].get_reshaped_view<const Pack**>();
```

Comment:
Isn't the var name `cld_frac_tot`? Meaning, isn't there a `_` between `cld` and `frac`, since they are two words?

Reply:
Ah, you're right. For some reason I thought we agreed on using `_` to separate words, much like we do in the code (and I believe that's also the CF convention).

Then I don't think this needs to be addressed in here. Though I might want to bring this up again soon.